GameTracker-Core/Controller.cs

```diff
@@ -0,0 +1,80 @@
+﻿using GameTracker_Core.Models;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GameTracker_Core
+{
+    public class Controller
```

Comment:
`private` fields prefixed with `_` and without `get` and `set`
`public` methods should start with a capital letter.